README.md

```diff
@@ -186,7 +186,9 @@ There are two mailing lists for discussions:
 
   https://lists.ovirt.org/admin/lists/users.ovirt.org/
 
-The developers also hang out on IRC at #vdsm hosted on freenode.net
+The developers also hang out on IRC at
+[#vdsm](irc://irc.freenode.org/vdsm) hosted on freenode.net
+([logs](http://ekohl.nl/vdsm)).
```

Comment:
#vdsm channel is not used for years, and nobody uses freenode now. The link should be replaced by #ovirt on irc.oftc.net.